### ISSUE TITLE
Delete tmp photo after photo storage

### ISSUE THREAD

**Issue by frankrousseau** [cozy/cozy-photos-v2#2]
Actually, when a photo is uploaded, the application saves it temporary in the tmp folder. Once all operations are done, an useless version of the photo is kept in the tmp folder.
It would be better to delete it.

**Comment by aenario** [comment]
During upload, the application use the ./uploads folder. A typo prevented delete from this folder (fixed in d9504d45)

However, the following piece of code also creates a file in the /tmp folder

``` coffeescript
Photo = require './server/models/photo.coffee'

photo =
    title: "Photo 1"
    rawpath: "./server/_specs/fixtures/test.jpg"

Photo.create photo, (err, photo) ->
    return res.error 500, "Creation failed.", err if err
    data = name: 'raw', type: "image/jpeg"
    photo.attachFile photo.rawpath, data, -> console.log "ok"
```

so it appears jugglingdb-cozy-adapter creates files in the /tmp folder. 

Is this an issue on linux ?

**Comment by frankrousseau** [comment]
No I think, that's the Data System that does not delete the file after receiving an attachment:
https://github.com/mycozycloud/cozy-data-system/blob/master/app/controllers/attachment_controller.coffee#L36-L55
